Infragraph LA docs update [IPE-1505] (#2710)
hashicorp/web-unified-docs: content/hcp-docs/content/docs/infragraph/connections/terraform-enterprise.mdx

To connect Infragraph to Terraform Enterprise, you must provide an API token. Refer to the [Team API Tokens](/terraform/enterprise/users-teams-organizations/api-tokens#team-api-tokens) documentation for more information.

## Set up the connection 

Complete the following steps to establish a new Terraform Enterprise connection:

1. From the Resource graph **Overview** at the top right, click **Create Connection**.
1. Select **Terraform Enterprise** from the list of data sources. 
1. Click **Authenticate Connection**. 
1. Under **TFE Hostname**, enter the address of your Terraform Enterprise instance. 
1. Follow the instructions to paste the team token.
1. Click the **Review connection details** button.
1. Review the connection information. 
1. Click the **Create connection** button.

The data source connects to your Terraform Enterprise organization. 

## Sync Terraform Enterprise data 

@include 'infragraph/sync-data.mdx'

## View Terraform Enterprise data  

To find Terraform Enterprise data connected to your resource graph, complete any of the following actions: 

- Go to the **Inventory** page. Click an individual resource type to view it in the graph explorer. 

> - Query resources with any of the existing queries that include Terraform Enterprise resources. 

Reviewer: ```suggestion
- Query resources with a saved query that includes Terraform Enterprise resources.
```
